### PR TITLE
Rename career-hero to hero-career to follow hero block guidelines

### LINKED ISSUE #15: Career testimonials: create hero block
Create hero block for career testimonials pages:

Screenshot 2023-09-14 at 13 09 25

### PR DESCRIPTION
**Pull request by bosschaert**
This fixes a layout problem with the hero block for career testimonials

Fixes #15

Test URLs:
- Before: https://main--sunstar--hlxsites.hlx.live/career/boedi-hartono
- After: https://issue-15-2--sunstar--hlxsites.hlx.live/career/boedi-hartono

### CODE REVIEW

File: blocks/hero-career/hero-career.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+function addTextEl(tag, txt, parent, ...classes) {
+  const newDiv = document.createElement(tag);
+  newDiv.textContent = txt;
+  classes.forEach((c) => newDiv.classList.add(c));
+  parent.appendChild(newDiv);
+}
+
+export default function decorate($block) {
```
Comment (sdmcraft) [inline blocks/hero-career/hero-career.js:20]: Does `$` carry a specific meaning here?

**Review by sdmcraft — APPROVED** [review]

File: blocks/hero-career/hero-career.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  const pic = createOptimizedPicture(photoURL, cfg.name);
+  heroLeft.append(pic);
+  addTextEl('h6', cfg.name, heroLeft, 'hero-career-name');
+  addTextEl('p', cfg.title, heroLeft, 'hero-career-title');
+
+  const heroRight = document.createElement('div');
+  heroRight.classList.add('hero-career-right');
+  heroDiv.appendChild(heroRight);
+  addTextEl('blockquote', cfg.quote, heroRight, 'hero-career-quote');
+  addTextEl('h6', 'Career background', heroRight, 'hero-career-careerbgtitle'); // TODO i18n
```
Comment (jindaliiita) [inline blocks/hero-career/hero-career.js:50]: how are we tracking these todos ?

File: blocks/hero-career/hero-career.js (hunk 1 above)
Comment (bosschaert) [reply blocks/hero-career/hero-career.js:20]: No, I just started this block originally from another block which had the `$` syntax. I'll remove it.

File: blocks/hero-career/hero-career.js (hunk 2 above)
Comment (bosschaert) [reply blocks/hero-career/hero-career.js:50]: We will have to have a separate effort (probably fairly small) to make these pages work with Japanese. This issue will become evident then.

**Pull request merged**